freqtrade/analyze.py

```diff
@@ -172,40 +191,42 @@ def get_signal(self, exchange: Exchange, pair: str, interval: str) -> Tuple[bool
         )
         return buy, sell
 
-    def should_sell(self, trade: Trade, rate: float, date: datetime, buy: bool, sell: bool) -> bool:
+    def should_sell(self, trade: Trade, rate: float, date: datetime, buy: bool,
+                    sell: bool) -> Tuple[bool, SellType]:
         """
         This function evaluate if on the condition required to trigger a sell has been reached
         if the threshold is reached and updates the trade record.
         :return: True if trade should be sold, False otherwise
         """
         current_profit = trade.calc_profit_percent(rate)
-        if self.stop_loss_reached(current_rate=rate, trade=trade, current_time=date,
-                                  current_profit=current_profit):
-            return True
+        stoplossflag = self.stop_loss_reached(current_rate=rate, trade=trade, current_time=date,
+                                              current_profit=current_profit)
+        if stoplossflag[0]:
```

Comment:
Do you think it makes sense to extract this tuple to a `NamedTuple`? The index syntax makes it hard to wrap your head around

Reply:
i was thinking about it but wasn't sure if it was worth if it's only a 2-value tuple.

changed it now :+1: